UnitTests/GitUITests/CommandsDialogs/FileStatusListTests.cs

```diff
@@ -265,9 +268,14 @@ public void Test_StoreFilter_valid(string regex, bool active)
             var expectedColor = active ? accessor.ActiveInputColor : SystemColors.Window;
             string expectedRegex = string.IsNullOrEmpty(regex) ? null : regex;
 
-            accessor.StoreFilter(regex);
+            _fileStatusList.SetFilter(regex);
 
             CheckStoreFilter(expectedColor, expectedRegex, accessor);
+
+            // FilterVisibleInternal has been reset because no items were added, toggle it
+            _fileStatusList.FilterVisible = false;
+            _fileStatusList.FilterVisible = true;
```

Comment:
Shouldn't these two lines be before the "act" phase (line:271)?
After we set a filter - the button should be visible

Reply:
Then the file list must be filled which is little too much for these test cases.
I tried to explain it with the comment `// FilterVisibleInternal has been reset because no items were added, toggle it`.